lib/Stripe/ApiRequestor.php

```diff
@@ -2,6 +2,26 @@
 
 class Stripe_ApiRequestor
 {
+    public static $CURLOPT_CONNECTTIMEOUT = 3;
```

Comment:
It is completely configurable (see Stripe_ApiRequestor::setTimeouts), and this can be call from anywhere in API land.